Assets/Scripts/Models/Furniture.cs

```diff
@@ -602,6 +737,10 @@ public void CancelJobs()
         }
     }
 
+    /// <summary>
+    /// Checks whether the furniture is a stockpile
+    /// </summary>
+    /// <returns>True is the furniture is a stockpile</returns>
     public bool IsStockpile()
```

Comment:
We should probably remove this function since is very tied to a single type of furniture.